olp-cpp-sdk-core/src/http/Network.cpp

```diff
@@ -35,18 +34,18 @@
 namespace olp {
 namespace http {
 
-CORE_API std::unique_ptr<Network> CreateDefaultNetwork(size_t max_requests_count) {
+CORE_API std::shared_ptr<Network> CreateDefaultNetwork(size_t max_requests_count) {
 #ifdef NETWORK_HAS_CURL
   CORE_UNUSED(max_requests_count);
```

Comment:
Define only once the CORE_UNUSED() please outside of the #ifdefs
You can still use the variable even if you mark it as unused.

Reply:
Done